packages/firebase/externs/firebase-firestore-externs.js

```diff
@@ -755,7 +755,7 @@ firebase.firestore.DocumentSnapshot.prototype.metadata;
 
 /**
  * An object containing all the data in a document.
- * @type {Object}
+ * @typedef {Object}
```

Comment:
I think this was also done in https://github.com/firebase/firebase-js-sdk/pull/3713